components/logo/index.scss

```diff
@@ -33,6 +34,6 @@
 .rvo-logo__subtitle {
   margin-block-end: 0;
   margin-block-start: 0;
-  max-width: 200px;
+  max-width: 220px;
```

Comment:
Misschien is het handig hier een token van te maken?

```css
max-inline-size: var(--rvo-logo-subtitle-max-inline-size, 220px);
```

Reply:
Goede suggestie! Ik heb het aangepast.